test/unittests/typed_value_test.cpp

```diff
@@ -0,0 +1,75 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "test/utils/typed_value.hpp"
+#include <gtest/gtest.h>
+
+using namespace fizzy;
+using namespace fizzy::test;
+
+TEST(typed_value, construct_contexpr)
+{
+    constexpr TypedValue i32{int32_t{-1}};
```

Comment:
Nit: some value literals are defined with `{}` others with `()`.

Are `_i32` literals merged yet? Can't we use them here?

Reply:
Ah I got it, it was intentional, because somewhere conversion is happening.